requirements.txt

```diff
@@ -1,6 +1,13 @@
-Django==1.10
+astroid==2.0.4
+Django==2.1
 django-cors-middleware==1.2.0
-django-extensions==1.7.1
-djangorestframework==3.4.4
+django-extensions==2.1.2
+djangorestframework==3.8.2
+isort==4.3.4
+lazy-object-proxy==1.3.1
+mccabe==0.6.1
 PyJWT==1.4.2
+pylint==2.1.1
```

Comment:
Make sure to include only necessary requirements in this file.... stuff like pylint should not be here.

Reply:
Unnecessary third party packages have been removed